Neos.Flow/Migrations/Mysql/Version20180827132710.php

```diff
@@ -2,6 +2,7 @@
 namespace Neos\Flow\Persistence\Doctrine\Migrations;
 
 use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Platforms\MySQLPlatform;
```

Comment:
Nice, but not used futher down, this still uses `$this->connection->getDatabasePlatform()->getName() != 'mysql'`